ampersand-filtered-subcollection.js

```diff
@@ -356,9 +356,8 @@ assign(FilteredCollection.prototype, Events, {
         }
 
         //If we were asked to sort, or we aren't gonna get a sort later and had a sortable property change
-        if (
-            action === 'sort' ||
-            (propName && !sortable && includes([this.comparator, this.collection.comparator], propName))
+        if (action === 'sort' ||
+            (propName && !sortable && includes([this.comparator, this.collection.comparator].concat(this._watched), propName))
```

Comment:
I don't think you need this concat here, I think you can just add `this._watched` to the array that is being checked against propName